tests/conftest.py

```diff
@@ -137,63 +137,95 @@ def setup_server(oidc_mock_idp: MockIdP):
 
     # Build podman container
     subprocess.check_call(
-        ["podman", "build", "-t", "nginx-webdav", "nginx", "-f", "nginx.dockerfile"]
+        ["podman", "build", "-t", "nginx-webdav", "nginx", "-f", "nginx/nginx.dockerfile"]
     )
 
-    yield
-
-    # Clean up
-    os.remove("nginx/lua/config.json")
-
-
-@pytest.fixture(scope="module")
-def nginx_server(setup_server) -> Iterator[str]:
-    """A running nginx-webdav server for testing
+    # Find an available port. 
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    open_port = 8280
+    for port in range(open_port,open_port+100):
+        try:
+            ret = self.sock.create_connection("localhost", port)
+        except:
+            # People have complaints about bare except, but I don't care
+            open_port = port
+            break
 
-    It's nice to have a module-scoped fixture for the server, so we can
-    reduce the number of irrelevant log messages in the test output.
-    """
     # Start podman container
     podman_cmd = [
         "podman",
         "run",
         "-d",
         "-p",
-        "8080:8080",
+        f"{open_port}:8080",
+        "-v",
+        "./nginx/conf.d:/etc/nginx/conf.d",
+        "-v",
+        "./nginx/lua:/etc/nginx/lua",
```

Comment:
After #22 I removed the bind mounts during testing, idea being that it then tests what is built in the CD pipeline

Reply:
Ah, makes sense, I'll respin